dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -3626,10 +3626,13 @@ void Spdp::SpdpTransport::on_data_available(DCPS::RcHandle<DCPS::InternalDataRea
 
   network_interface_address_reader_->take(samples, infos);
 
+  ACE_Reactor* reactor_ptr = reactor();
```

Comment:
I would move this check to the `MulticastManager::process` method.
Somethink like
```
if (reactor) {
  register_handler
}
```
so that the rest of the processing happens.

Reply:
> Do you have the rest of the threads? Was this start up or shutdown?

I don't, but I remember this wasn't shutdown... somewhat early in the Thrasher output, but not immediate (maybe one or two transports had been configured?) I will put an assert in the code and try to get a better picture of why it's happening at all.